tarynmartin/shopify-challenge
# Displaying search results in results section

#### What’s this PR do?
- Creates cards to display each individual movie's info
- Iterates through returned movies and displays MovieCards in results
- Associated styling

#### Where should the reviewer start?
- Results.js to see how cards are being displayed
- MovieCard.js to see how cards are displayed
- Associated CSS files to see styling

#### How should this be manually tested?
- in terminal, run npm start
- in browser, do a search to see search results displayed

#### What are the relevant tickets?
resolves #3 resolves #4
